src/index.html

```diff
@@ -1,13 +1,70 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body class="page">
+
+    <div class="card" data-qa="card">
+      <div class="product">
+        <div class="product__image">
+          <img
+            src="./images/imac.jpeg"
+            alt="Photo iMac"
+            class="image"
+          >
+        </div>
+        <div class="product__description">
+          <section class="title">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </section>
+          <section class="article">
+            Код товара: 195434
+          </section>
```

Comment:
Мне кажется это не секции, но не уверен, могу апрувнуть если не хочешь менять
```suggestion
          <h2 class="product__name">
            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
          </h2>
          <p class="product__code">
            Код товара: 195434
          </p>
```